Dockerfile

```diff
@@ -15,12 +15,12 @@ ENV PYTHONUNBUFFERED=1
 
 COPY --from=builder /app .
 
-RUN pip install --upgrade pip && \
-    pip install -r requirements.txt
+#RUN pip install --upgrade pip && \
+#    pip install -r requirements.txt
```

Comment:
The `RUN` command for installing dependencies is commented out. This is necessary to ensure that all required Python packages are installed in the container. Please uncomment and ensure the `requirements.txt` file is correctly specified.